src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -82,6 +83,9 @@
     <Compile Include="System\WindowsRuntimeSystemExtensions.CoreRT.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNETCoreApp)'=='true' or '$(TargetsUAP)'=='true'">
+    <Compile Include="$(CommonPath)\CoreLib\System\WinRTFolderPaths.cs">
```

Comment:
Should this file be moved to somewhere under  src/System.Runtime.WindowsRuntime/src ?